02-social-links/style.css

```diff
@@ -0,0 +1,223 @@
+* {
+  margin: 0;
+  padding: 0;
+  /* box-sizing: content-box; */
+  font-family: "Inter", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 500;
+  font-style: normal;
+  font-variation-settings: "slnt" 0;
+  transition: all 0.2s;
+}
+article {
+  display: flex;
+}
+a {
+  text-decoration: none;
+  color: inherit;
+  font-weight: 700;
+  font-size: 0.9rem;
+}
+.logo {
+  display: block;
+  width: 100px;
+  border-radius: 50%;
+  margin: 0.5rem auto;
+  margin-bottom: 1.5rem;
+}
+li {
+  list-style: none;
+  background-color: hsl(0, 0%, 20%);
+  color: hsl(0, 0%, 100%);
+  padding: 0.8rem 2.5rem;
+  border-radius: 0.5rem;
+  width: 15rem;
+}
+li:hover {
+  background-color: hsl(75, 94%, 57%);
+  color: hsl(0, 0%, 8%);
+}
+ul {
+  gap: 1rem;
+  display: flex;
+  flex-direction: column;
+  margin-top: 1rem;
+}
+
+/* a {
+  display: grid;
+  width: 100%;
+  margin: 0.5rem auto;
+  padding: 1rem;
+  border: none;
+} */
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  background-color: #141414;
+}
+
+/* .container {
+  width: 30%;
+  background-color: #1f1f1f;
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+} */
+.container {
+  background-color: hsl(0, 0%, 12%);
+  color: hsl(0, 0%, 100%);
+  display: flexbox;
```

Comment:
Typo. Must be `flex`, not `flexbox` :)

Also, I noticed you are overusing flexbox. Everything is `display: flex`. Everything doesn't have to be a flexbox.